docs/Getting Started/Integration Basics - Java/index.md

```diff
@@ -764,3 +764,6 @@ The `SDLSessionBean` should be inside a Java package other than the default pack
 * Create Manifest. Apply + OK.
 * Run Build -> Build Artifacts to get a .war file in the /out folder.
 !@
+
+## Where to Go From Here
+You should now be able to connect to a head unit or emulator. From here, [learn about designing your main interface](Displaying a User Interface/Main Screen Templates). For further details on connecting, see [Connecting to a SDL Core](Getting Started/Connecting to an Infotainment System). To learn optimal design patterns for SDL checkout the [Best Practices guide](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/).
```

Comment:
```suggestion
```suggestion
You should now be able to connect to a head unit or emulator. From here, [learn about designing your main interface](Displaying a User Interface/Main Screen Templates). For further details on connecting to a test module or emulator, see [Connecting to a SDL Core](Getting Started/Connecting to an Infotainment System). To learn about best practices for an SDL app, see our [Best Practices guide](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/).
```
```